### PR TITLE
feat: supports configuring multiple converters

### PR DESCRIPTION
**Pull request by qeesung**

### CODE REVIEW

**Comment by qeesung** [comment]
@jimlambrt Would you please spend a few precious minutes reviewing this Pull Request?

File: options.go [hunk 1]
```diff
@@ -67,8 +67,7 @@ func WithConverter(fieldName string, fn ValidateConvertFunc) Option {
 	return func(o *options) error {
 		switch {
 		case fieldName != "" && !isNil(fn):
-			o.withValidateConvertFn = fn
-			o.withValidateConvertColumn = fieldName
+			o.withValidateConvertFns[fieldName] = fn
```
Comment (jimlambrt) [inline options.go:70]: should we prevent duplicate converters?  for example, if someone specifies two
converters for the `name` field?

**Review by jimlambrt — CHANGES_REQUESTED** [review]

File: options.go (hunk 1 above)
Comment (qeesung) [reply options.go:73]: Added duplicate detection, if the user specifies a converter for the same field multiple times, an error will be returned

**Review by jimlambrt — APPROVED** [review]
Ty for the contribution!

**Pull request merged**